auditlog/context.py

```diff
@@ -62,6 +66,7 @@ def _set_actor(user, sender, instance, signal_duid, **kwargs):
             and instance.actor is None
         ):
             instance.actor = user
+            instance.actor_email = hasattr(user, "email") and user.email or None
```

Comment:
:thinking: this is a bit cryptic, why not:

`getattr(user, "email", None)`

Reply:
feel free to riase a PR and change it